.travis.yml

```diff
@@ -64,4 +64,4 @@ deploy:
     local_dir: doc/compodoc
     github_token: $GH_REPO_TOKEN
     on:
-      branch: master
+      branch: compodoc
```

Comment:
won't that prevent updates of the docs from the master branch, e.g. when in the future a PR is adding new code as well as the related documentation to the master?

Should this maybe become 
```
branch:
  - master
  - compodoc
```
?

Reply:
yeah we tried to do something like this (@TheSlimvReal  is the expert concerning travis)...
unfortunately that didn't work (syntactically a string is required) and writing 'branches: [...] ' fixed that issue syntactically (no error is thrown), but instead, the command is never executed..
So my plan was to keep 'compodoc' for now so that I can look at the outcome without merging into master all the time. I see no big issue there (functionality/...) and added changing it back later to my todo in the issue #268 . If you don't like that solution we could also leave out that condition and override the compodoc on each branch (leads to far more inconsistency).

Reply:
ah, I see. Seems like you can set multiple branches in `on:` but a regex could be a workaround:

```
on:
  all_branches: true
  condition: ${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH} =~ ^(master|compodoc)$
```
https://stackoverflow.com/a/48959539/1473411

Reply:
Ok, I just used your code snippet, it seems to do the right thing. 
I didn't know that you could use regex there. Thanks.

I'd still be interested if this works as well:
```if: branch IN (compodoc, master)``` (https://docs.travis-ci.com/user/conditions-v1)